app.js

```diff
@@ -1,13 +1,24 @@
 const express = require('express');
 const app = express();
-const { getTopics } = require('./controller')
+const { getTopics, getArticles } = require('./controller')
 
 app.use(express.json());
 
 app.get('/api/topics', getTopics);
 
+app.get('/api/articles', getArticles);
+
+app.all('*', (request, response, next) => {
+    response.status(404).send({msg: 'Not found!'});
+});
+
 app.use((err, req, res, next) => {
-    console.log(err);
+    console.log(err.status)
+    if (err.status === 404) {
+        res.status(err.status).send({msg : "Not Found!"})
+    } else {
+        next(err)
+    }
 })
```

Comment:
These are currently doing the same thing - returning 404. I think your app.use should be a 500 internal server error